src/sentry/integrations/slack/requests.py

```diff
@@ -119,9 +119,9 @@ def _check_signing_secret(self, signing_secret):
         signature = self.request.META["HTTP_X_SLACK_SIGNATURE"]
         timestamp = self.request.META["HTTP_X_SLACK_REQUEST_TIMESTAMP"]
 
-        req = six.binary_type("v0:%s:%s" % (timestamp, self.request.body))
-        request_hash = "v0=" + hmac.new(six.binary_type(signing_secret), req, sha256).hexdigest()
-        return hmac.compare_digest(six.binary_type(request_hash), six.binary_type(signature))
+        req = u"v0:{}:{}".format(timestamp, self.request.body).encode("utf-8")
```

Comment:
This looks super fishy. Why would `request.body` be unicode?

Reply:
I would have assumed this would have to be this:

```python
req = b"v0:%s:%s" % (timestamp.encode("utf-8"), self.request.body)
```

Reply:
Yeah, looking again I think you're right, body should hopefully be bytes. 😬 

This is probably ending up something like `v0:...:b'body value'` in py3 land

Reply:
The body in tests was explicitly not a byte string, I fixed the test body to be of type `bytes` to be consistent with what django gives back.